nominees/openattestation.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "OpenAttestation",
+  "aliases": [
+    ""
+  ],
+  "description": "OpenAttestation is an open-sourced framework that allows for trustworthy education data-sharing by facilitating the decentralised issuance, storage and verification of academic and training documents, in line with applicable data privacy regulations.",
+  "website": "https://www.openattestation.com/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/Open-Attestation/open-attestation/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 4,
+      "evidenceText": "Distributed-ledger technologies are a decentralized, transactional database technology that facilitate validated and peer-to-peer transactions that are consistent across a large number of anonymous network participants - sometimes referred to as nodes. Given its decentralized and tamper-resistant nature, distributed-ledger technology comes with promises of tackling issues of information traceability, accountability, and integrity among a wide variety of individuals, organizations, and institutions across society while improving privacy, transparency, robustness, trustworthiness, and authenticity.   The applications of emerging technology have been proliferating in various sectors and, among them, the education field. Distributed-ledger technology can be harnessed to address challenges to issue and verification of educational data. OpenAttestation is an open-sourced framework that supports the issuance, storage, and verification of different types of academic information, and complies with applicable regulations, protecting the privacy of users’ personal data, all these benefits align with and make contributions to SDG 4: Quality Education. ",
+      "evidenceURL": "https://www.openattestation.com/"
```

Comment:
I don't see any education use cases on the open attestation website, though I think there may be applicability to SDG 4. I think there are other SDG-related use cases for this platform, particularly related to governance (SDG 16). Would be interested to hear what @jwflory thinks.

Reply:
Resolved in commit 180f819